testfiles/1ZoneUncontrolled3SurfaceZone.idf

```diff
@@ -0,0 +1,397 @@
+!-Generator IDFEditor 1.34
+!-Option OriginalOrderTop UseSpecialFormat
+!-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
+!-      Use '!' comments if they need to be retained when using the IDFEditor.
+!1ZoneUncontrolled3SurfaceZone.idf
```

Comment:
It was already in this branch, so you should delete it here, and add it to EnergyPlusDevSupport\DefectFiles\5261 instead.  But I'm concerned that we may need to post a new issue since this doesn't converge for warmup.